### DISCUSSION TITLE
我觉得这个产品的思路有点问题，建议产品多考虑一下专业行业的特定问题解答思路

### DISCUSSION THREAD

**Discussion by ssrsybz** [InternLM/MindSearch#85] [Announcements]
提出一个好问题，答案就解决一半。
在问题的分析上，咱们的分析策略应该做成用户可以自己填写或者选择。
也就是拆分问题的时候，直接让用户进行参与。
因为很多人没办法把问题描述清楚。但是真正的核心问题，是可以被引导出来的。
所以我的第一建议是，把用户的问题进行拆分后，让用户参与选择他需要的哪些纬度。

就比如，我在写自媒体的口播稿子时，我需要尝试从不同的角度来变换我提问的问题。以保证我的这个问题上的关键词，能涵盖到搜索引擎上的关键词。
通常变换的问句，确实能够提供新的搜索结果，而我就可以用这种近似的结果来相互佐证内容是否是一致。

但是通常我要完成稿子时，需要我问到非常多具体的问题，我才能将我的稿子写的内容详实。
比如我有一期视频讲医疗信息化的发展历史，我最开始的搜索问句是，：医疗信息化发展历史、医疗软件、医院信息化、医生工作站、医院HIS系统，通过这些关键词，我找到了大量的企业发布的营销文。你看到的是他们把关键词罗列到这些文章上，这些罗列到关键词，你根本无法和发展历史结合一起的。
然后我的搜索策略就变成了，我需要找到一个类似教科书的东西，把医院信息系统从那一年发展的、发展了什么、出了什么政策、有什么软件公司参与、产品形态是什么，一年一年的找出来。
这种就在教科书、医院信息化简史、一文读懂医院信息软件，这些关键词里可以看到。

在这些书籍和文章里，我会针对感兴趣的内容，继续在搜素引擎里深挖。并且我在这里加入了时间的纬度。
即，医疗信息化发展+2001年至2002年，这种格式。
这个格式的好处在于，2000年百度才出来，至少在那几年，能写这种文章的人，写的东西言之有物。
同时搜索出来的内容，天然的会有当时的特色。而且这些内容里，不会出现未来的产品和公司。我就能在时间线上，把产品、市场的发展逻辑理清楚。

因为我这个问题，想要知道的是医疗信息化行业的发展。在各种关键词和结果的引导下，自然而然的就会发现行业里的头部垂直网站叫什么，在垂直网站里，确实有很多高质量、垂直的内容。但是也因为网站开发的早，里面的搜索引擎是非常差的。
十几家的垂直网站，提供了丰富的内容，当然这些网站都是在2012年之后建立的。具体原因和互联网发展节点有关。

然后我为了让整理出来的内容，更具有真实性，我还在图书馆网、期刊网、医疗信息网，类似这种网站上，寻找当时信息科主任发布的登报文章。在通过文章里的引用材料看自己整理的内容是否有偏差。

最后的一步就是，在政府网站上，我需要知道最近20年的政策，如何引导市场的发展。
因为在医院这个行业里，政府政策占绝对的主导。政策落地的时间和内容，对医院信息产品的影响也巨大。往往政策在制定时，就是各大行业头部公司的行业专家去协作的。所以分析企业产品的发布时间和政策的发布时间，就能得到比较有趣的观点。

上面是我在做一个深度内容时的搜索流。
讲这些的意思是，你的产品既然想到了要先思考问题是什么，那么其实是可以让我自己来填写我是怎么思考这些问题的。
当然这个填写或者是选择的方法，产品设计上不难，技术实现上有待考证。

我第二个想要讨论的点是，在垂直领域里做深度搜索，比现在同质化做广度搜索要更有钱景。
上面讲到我在制作深度内容，并且以视频形式发布出去，在视频一发布的时候，我就获得了大量的粉丝关注。粉丝里面的构成和我当时预想的一样。有教这个行业的老师、在学这个行业的学生、应用这个软件的医院、给医院开发软件的公司老板、员工和想要在这个行业进行创业的创业者。这让我觉得，内容搜索或者说内容整理是一个有非常大的钱景。
因为我们的社会里发展的太快了，稍微没在注意到网络上的新闻内容，过后你就很难在找到。然后就会有我这样把信息差给抹平的人，帮助人们来了解、学习这个行业。
也就是我，我完全可以通过搜索，将一个我不懂的行业，变成我很懂的行业，然后通过我的二次讲解，让这个行业的人认为我是行业专家。
在我和粉丝互动的过程中，粉丝的更多需求在于，想要我手上的资料。准确的说，想要我已经整理好的结构化的资料。
这对于我来说，就变成了可以持续盈利的点。知识付费嘛，一份资料多少钱，一个社群多少钱。
但是我知道，最有价值的是我对于这个事情的搜索思路，而不是我搜索了多少资料。

然后我就尝试了很多大模型的搜索，kimi、星火、ChatGPT，这些确实有提高效率，但是并不能让我把搜索思路更好的展示给我的学员、粉丝。
直到最近发现了秘塔的搜索，我感觉搜索似乎有那个味道了
但是这仍然是将一个问题的答案，直接输出给我，只是结构化了答案的来源。
而我看到你的产品，我觉得你的方向是对的，而且在落地垂直行业上，已经有了初步雏形。

如果让我作为产品经理去改，我就会往B端垂直行业上去优化产品。
针对问题所属的行业，问题的形变，问题的延伸做一个思的自定义调整过程。
然后针对答案来源的网站、资料在进行一个自定义的总结的过程。
当把一个问题调通之后，就可以针对现有的各个行业出专业版搜索。

**Comment by Harold-lkk** [comment]
说的很深刻，这也是我们想发展的方向之一，就是怎么提供`这个事情的搜索思路` 怎么让模型知道或者通过辅助知道专业人士的`思维方式`。这个涉及到模型要从开始从模拟人说话到模拟人思考的过程，对齐到专业人士的思考方式，思考路径或者说学会你这个`深度思考`的逻辑。

**Reply by ssrsybz** [reply]
我觉得这个方向是大语言模型商业化的最好方向。今天发现秘塔上线了法律文书的结构化搜索。这就和我想的差不多了，先把不同纬度的内容存起来，法律条文是一类，案例是一类，再根据用户提问的问题去条文和案例里找关联内容。如果提问不够准确，可以在提问上进行问题联想，这样就能做到既能看到结果，又能改进思考方向。从我们的产品上，我觉得有两个点可以做，一个是对知识进行结构化的存。二是对问题进行结构化的联想。基于我们现在项目，我觉得增加一个指定网站搜索的功能，除了搜索，还能整站存下来。第二个是，增加特定行业关键词、关键短语的人工配置页面。比如医院信息化这个词，下面应该有非常多的二级词汇，可以从前面保存的整站内容里提取出来，这样用户在进行提问时，基于整站内容，就能得到比较准确的索引。这就可以做成一个最小可用的行业垂直类产品。而且商业化非常容易！对这个项目，希望能和你深入探讨一下。
